moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -160,8 +184,7 @@ void RobotModelLoader::configure(const Options& opt)
   if (model_ && opt.load_kinematics_solvers_)
     loadKinematicsSolvers();
 
-  ROS_DEBUG_STREAM_NAMED("robot_model_loader", "Loaded kinematic model in " << (ros::WallTime::now() - start).toSec()
-                                                                            << " seconds");
+  RCLCPP_DEBUG(node_->get_logger(), "Loaded kinematic model in %d seconds", (clock.now() - start).seconds());
```

Comment:
`rclcpp::Time` is much cleaner than using `std::chrono` directly. We should apply this to other packages as well as long as there is no reason against it.